DynamicData/List/Internal/TransformMany.cs

```diff
@@ -50,6 +50,28 @@ internal sealed class TransformMany<TSource, TDestination>
         {
         }
 
+        private static ReadOnlyObservableCollection<TDestination> CreateCollectionFromList(IObservableList<TDestination> list)
```

Comment:
I agree with your comment on the PR. There must be a cleaner way to achieve the goal, without any need to use bind.

Reply:
I was afraid so ;-) Does the proposal above make sense? Happy to spend some time on this, but since I'm relatively new to this library and Rx in general, I could use a little bit of help :-) Thanks!

Reply:
I will get involved on this one as there is clearly a missing transform many overload for both list and cache parts or the system. The import thing is not to resolve the issue quickly but to find a good solution with good performance characteristics

Reply:
Makes sense, thanks for that! If there's anything I can do, let me know.